app/main.py

```diff
@@ -1,6 +1,20 @@
-def get_weather() -> None:
-    # write your code here
-    pass
+import os
+
+import requests
+from dotenv import load_dotenv
+
+load_dotenv()
+URL = "http://api.weatherapi.com/v1/current.json"
+city = "Paris"
```

Comment:
According to the checklist, constants should be in uppercase style. The `city` variable is a constant in your code, so it should be renamed to `CITY`.